packages/ng-primitives/tooltip/src/tooltip-text-content/tooltip-text-content.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component } from '@angular/core';
+import { injectOverlayContext } from 'ng-primitives/portal';
+import { NgpTooltip } from '../tooltip/tooltip';
+
+
+/**
+ * Internal component for wrapping string content in tooltip portals
+ * @internal
+ */
+@Component({
+  template: '{{ content() }}',
+  hostDirectives: [NgpTooltip],
+  host: {
+    ngpTooltip: '',
```

Comment:
I can add a comment here that it's for styling targeting purposes as the host directive isn't actually being added at the DOM and this is our entry point for styling.